Contributing.md

```diff
@@ -0,0 +1,18 @@
+# Contributing
+Follow these steps to add a function to this library.
+
+## General
+1. Clone the repo
+1. Run `npm install` in the repo
+1. Find [an issue]() to contribute to
```

Comment:
Is this intended to be a link to this repo's issues?

Reply:
oh yeah

Reply:
@the-pat should be fixed now.